Documentation/6.0.2/README.rst

```diff
@@ -0,0 +1,1515 @@
+
+
+
+Phase 8. Nuage Docker Containers
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
+1. On the Undercloud, use the following instructions to get Nuage images from a Red Hat container registry using registry service account tokens. You will need to `create a registry service account <https://access.redhat.com/terms-based-registry>`_ to use prior to completing the following task.
+
+::
+
+    $ docker login registry.connect.redhat.com
+    Username: ${REGISTRY-SERVICE-ACCOUNT-USERNAME}
+    Password: ${REGISTRY-SERVICE-ACCOUNT-PASSWORD}
+    Login Succeeded!
+
+2. Now change the working directory to /home/stack/nuage-tripleo-heat-templates/scripts/pull_nuage_containers/
+
+::
+
+    $ cd /home/stack/nuage-tripleo-heat-templates/scripts/pull_nuage_containers/
+
+3. Configure `nuage_container_config.yaml` with appropriate values and a sample is given below.
+
+::
+
+    #OpenStack version number
+    version: 13
+    #Nuage Release and format is <Major-release, use '-' instead of '.'>-<Minor-release>-<Updated-release>
+    # for exmaple: Nuage release 6.0.2 please enter following
+    release: 6-0-2
+    #Tag for Nuage container images
+    tag: latest
+    #Undercloud Local Registry IP Address:PORT
+    local_registry: 192.168.24.1:8787
+    #List of Nuage containers
+    nuage_images: ['heat-api-cfn', 'heat-api', 'heat-engine', 'horizon', 'neutron-server', 'nova-compute']
+
+4. Now execute `nuage_container_pull.py` script by passing nuage_container_config.yaml to "--nuage-config" argument.
+
+::
+
+    $ python nuage_container_pull.py --nuage-config nuage_container_config.yaml
+
+5. The above command does the below four steps:
+
+:Step1: Pull Nuage container images from Red Hat Registry
+
+:Step2: Retag the Nuage container images, by modifying the registry to point to local registry
+
+:Step3: Push the retagged Nuage container images to local registry
+
+:Step4: Remove the container images that got created in step1 and step2 from undercloud machine.
+
+6. After executing `nuage_container_pull.py`, there will be a nuage_overcloud_images.yaml created under /home/stack/nuage-tripleo-heat-templates/environments and always /home/stack/templates/overcloud_images.yaml should take precedence over this file.
+
+::
+
+    Ex:
+    openstack overcloud deploy --templates -e /home/stack/templates/overcloud_images.yaml -e /home/stack/nuage-tripleo-heat-templates/environments/nuage_overcloud_images.yaml - e <remaining environment files>
+
+
+Phase 9: Deploy the Overcloud
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can use the Heat templates with the the command-line based template to deploy the Overcloud.
```

Comment:
Remove the extra "the" before "command-line based template":
You can use the Heat templates with the the command-line based template to deploy the Overcloud

Reply:
done